WordPress/Classes/ViewRelated/Menus/Views/MenuItemSourceCell.m

```diff
@@ -0,0 +1,165 @@
+    if (self) {
+        
+        self.backgroundColor = [UIColor whiteColor];
+        
+        [self initStackView];
+        [self initLabelsStackView];
+        [self initLabel];
+    }
+    
+    return self;
+}
+
+- (void)initStackView
+{
+    UIStackView *stackView = [[UIStackView alloc] init];
+    stackView.translatesAutoresizingMaskIntoConstraints = NO;
+    stackView.distribution = UIStackViewDistributionFill;
+    stackView.alignment = UIStackViewAlignmentLeading;
+    stackView.axis = UILayoutConstraintAxisHorizontal;
+    
+    UIEdgeInsets margins = UIEdgeInsetsZero;
+    margins.top = 10.0;
+    margins.left = MenusDesignDefaultContentSpacing;
+    margins.right = MenusDesignDefaultContentSpacing;
+    margins.bottom = 10.0;
+    
+    stackView.layoutMargins = margins;
+    stackView.layoutMarginsRelativeArrangement = YES;
+    stackView.spacing = MenusDesignDefaultContentSpacing / 2.0;
+    [self.contentView addSubview:stackView];
+    
+    _leadingLayoutConstraintForContentViewIndentation = [stackView.leadingAnchor constraintEqualToAnchor:self.contentView.leadingAnchor];
+    _topLayoutDefaultConstraint = [stackView.topAnchor constraintEqualToAnchor:self.contentView.topAnchor];
+    _topLayoutConstraintForContentViewIndentation = [stackView.topAnchor constraintEqualToAnchor:self.contentView.topAnchor constant:-(margins.top)];
+    
+    [NSLayoutConstraint activateConstraints:@[
+                                              _topLayoutDefaultConstraint,
+                                              _leadingLayoutConstraintForContentViewIndentation,
+                                              [stackView.trailingAnchor constraintLessThanOrEqualToAnchor:self.contentView.trailingAnchor],
+                                              [stackView.bottomAnchor constraintEqualToAnchor:self.contentView.bottomAnchor]
+                                              ]];
+    
+    _stackView = stackView;
+}
+
+- (void)initLabelsStackView
+{
+    UIStackView *labelsStackView = [[UIStackView alloc] init];
+    labelsStackView.translatesAutoresizingMaskIntoConstraints = NO;
+    labelsStackView.distribution = UIStackViewDistributionFill;
+    labelsStackView.alignment = UIStackViewAlignmentTop;
+    labelsStackView.axis = UILayoutConstraintAxisHorizontal;
+    labelsStackView.spacing = self.stackView.spacing;
+    _labelsStackView = labelsStackView;
+
+    NSAssert(_stackView != nil, @"stackView is nil");
+    [_stackView addArrangedSubview:labelsStackView];
+}
+
+- (void)initLabel
+{
+    UILabel *label = [[UILabel alloc] init];
+    label.translatesAutoresizingMaskIntoConstraints = NO;
+    label.font = [WPStyleGuide tableviewTextFont];
+    label.textColor = [WPStyleGuide greyDarken30];
+    label.backgroundColor = [UIColor whiteColor];
+    label.numberOfLines = 0;
+    label.lineBreakMode = NSLineBreakByTruncatingTail;
+    
+    [label setContentHuggingPriority:UILayoutPriorityRequired forAxis:UILayoutConstraintAxisHorizontal];
```

Comment:
> Just curious, why less than or equal for this one?
Maybe it was because content hugging was maxed on the label? If so maybe just max the compression resistance to avoid unwanted truncation and relax content hugging. I think that would make more sense with the statck view's distribution set to fill.